media-gfx/gif2apng/gif2apng-1.9.recipe

```diff
@@ -0,0 +1,50 @@
+SUMMARY="Converts GIF animations into APNG format"
+DESCRIPTION="This program converts GIF animations into animated PNG format. \
+Usually it makes the files smaller."
+HOMEPAGE="http://gif2apng.sourceforge.net/"
+COPYRIGHT="2009-2013 Max Stepin"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/gif2apng/gif2apng-$portVersion-src.zip"
+CHECKSUM_SHA256="3b21308e935d799b3ffb4a86c6e00ffa4cb9b3f72f52d58d51c66eb0574ae7d2"
+SOURCE_DIR=""
+
+ARCHITECTURES="x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	gif2apng$secondaryArchSuffix = $portVersion
+	cmd:gif2apng = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:automake
+	cmd:ld$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	make
+}
+
+INSTALL()
+{
+	if [ "$targetArchitecture" != x86_gcc2 ]; then
+		commandBinDir=$binDir
+	else
+		commandBinDir=$prefix/bin
+	fi
+	echo $commandBinDir
```

Comment:
It would be better without the echo $commandBinDir ;-)

Everything else looks OK. I'll build it on each of my architectures and then merge.